Dockerfile

```diff
@@ -10,15 +10,16 @@ LABEL maintainer="hackerman"
 RUN \
  echo "**** install build packages ****" && \
  apk add --no-cache --virtual=build-dependencies \
-	git \
-	composer && \
+	git && \
  echo "**** install runtime packages ****" && \
  apk add --no-cache \
+	composer \
```

Comment:
Does lychee use composer after the init setup?

Reply:
No, but removing it results in a bunch of php modules to also be removed, some of which may be necessary. I'm too lazy to test each one. It's only 3mb unpacked

Reply:
Fair enough